cmd/version.go

```diff
@@ -57,6 +57,8 @@ func printJSONVersion() {
 	gaugeVersion := versionJSON{version.FullVersion(), version.CommitHash, make([]*pluginJSON, 0)}
 	allPluginsWithVersion, err := pluginInfo.GetAllInstalledPluginsWithVersion()
 	if err != nil {
+		// logger can not be used, since it breaks the json format,
```

Comment:
I had to fix this for getgauge/gauge-vscode#431. Raised https://github.com/getgauge/gauge/pull/1651

Reply:
Done